core/src/main/c/share/txn_board.cpp

```diff
@@ -164,13 +163,15 @@ class txn_scoreboard_t {
     void init(uint32_t entry_count) {
         mask = entry_count - 1;
         size = entry_count;
-        int64_t expected = 0;
-        // Since txn values are guaranteed to be greater than 0, min can be 0 only on
-        // a newly created scoreboard. So, this CAS should only succeed single time.
-        min.compare_exchange_strong(expected, L_MIN);
```

Comment:
TBH I have no idea why we were using CAS operation here considering that this method is called on txn scoreboard initialization, so no one mutates it concurrently.